rapids-4-spark-tools/src/test/resources/log4j.properties

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright (c) 2019, NVIDIA CORPORATION.
```

Comment:
Please update copyright year to 2021

Reply:
Changed to 2019-2021 since this is an existing file I copied from another module